python/example_code/ecs/create_cluster.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2010-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# This file is licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+
+import boto3
+try:
+  ecs_client = boto3.client('ecs', aws_access_key_id='ACCESS_KEY',
+                                   aws_secret_access_key='SECRET_KEY',
+                                   region_name='REGION')
+  response = ecs_client.create_cluster(clusterName='CLUSTER_NAME')
+  print(response)
+
+except BaseException as exe:
+    print(exe)
+#snippet-comment:[These are tags for the AWS doc team's sample catalog. Do not remove.]
+#snippet-sourcedescription:[create_instance.py demonstrates how to create an Amazon EC2 instance.]
+#snippet-keyword:[Python]
+#snippet-keyword:[AWS SDK for Python (Boto3)]
+#snippet-keyword:[Code Sample]
+#snippet-keyword:[AWS Identity and Access Management (IAM)]
+#snippet-service:[iam]
```

Comment:
Ther service needs to be updated
```suggestion
#snippet-keyword:[Amazon Elastic Container Service (Amazon ECS)]
#snippet-service:[ecs]
```